docs/basic-features/typescript.md

```diff
@@ -97,15 +97,27 @@ export default (req: NextApiRequest, res: NextApiResponse<Data>) => {
 
 ## Custom `App`
 
-If you have a [custom `App` ](/docs/advanced-features/custom-app), you can use the built-in type `AppProps` and change file name to `./pages/_app.tsx` like so:
+If you have a [custom `App`](/docs/advanced-features/custom-app), you can use the built-in type `AppProps` and change file name to `./pages/_app.tsx` like so:
 
 ```ts
-import { AppProps } from 'next/app'
+import App, { AppProps, AppContext } from 'next/app'
```

Comment:
```suggestion
import /* App, */ { AppProps, AppContext } from 'next/app'
```

Reply:
Should I do the same for `AppContext`?

Reply:
@rafaelalmeidatk yeah that'd be a good idea.